### ISSUE TITLE
selftesting fails on openSUSE

### ISSUE THREAD

**Issue by scarabeusiv** [pypa/virtualenv#1721]
When building updated package for 2.0.10 the test_create_no seed with some options fails on python3 (seem not to be run under py2):

```
[  158s] FAILED tests/unit/create/test_creator.py::test_create_no_seed[root-venv-copies-isolated]
[  158s] FAILED tests/unit/create/test_creator.py::test_create_no_seed[root-venv-copies-global]
[  158s] FAILED tests/unit/create/test_creator.py::test_create_no_seed[root-venv-symlinks-isolated]
[  158s] FAILED tests/unit/create/test_creator.py::test_create_no_seed[root-venv-symlinks-global]
[  158s] FAILED tests/unit/create/test_creator.py::test_create_no_seed[venv-venv-copies-isolated]
[  158s] FAILED tests/unit/create/test_creator.py::test_create_no_seed[venv-venv-copies-global]
[  158s] FAILED tests/unit/create/test_creator.py::test_create_no_seed[venv-venv-symlinks-isolated]
[  158s] FAILED tests/unit/create/test_creator.py::test_create_no_seed[venv-venv-symlinks-global]
```

Full build log: 
[buildlog.txt](https://github.com/pypa/virtualenv/files/4329103/buildlog.txt)

Also there are 4 build failures on both py2 and py3:
```
[   77s] FAILED tests/unit/create/test_creator.py::test_create_distutils_cfg[cpython2-posix]
[   77s] FAILED tests/unit/create/test_interpreters.py::test_failed_to_find_implementation[/usr/bin/python2]
[   77s] FAILED tests/unit/discovery/py_info/test_py_info.py::test_py_info_cache_clear
[   77s] FAILED tests/unit/discovery/py_info/test_py_info.py::test_py_info_cached_symlink
```

Full build log of that one (previous one has them skipped to see the other ones):
[buildlog.txt](https://github.com/pypa/virtualenv/files/4329122/buildlog.txt)

Any idea if there is something really borked or if it is safe to ignore these in our build enviroment?

**Comment by gaborbernat** [comment]
@scarabeusiv seems like an actual valid failure; do you have a docker image that replicates?

**Comment by scarabeusiv** [comment]
It is the rolling release one, Tumbleweed.
If you go for dockerfile the `FROM opensuse/tumbleweed` should be enough.

Also I have the package in OBS: https://build.opensuse.org/package/show/devel:languages:python/python-virtualenv

And if you can get your hands on 'osc' tool you can from commandline try to trigger the build and chroot to the virtual machine used by the OBS itself (sadly one needs to have user account to do that):

```
osc co devel:languages:python/python-virtualenv
cd devel:languages:python/python-virtualenv
osc build -M test
osc chroot
cd /home/abuild/rpmbild/BUILD/python-virtualenv*/
pytest-3.8 -vv
```

**Comment by gaborbernat** [comment]
@scarabeusiv see https://github.com/pypa/virtualenv/pull/1731 for a docker file that now passes, will merge and release right now

**Comment by scarabeusiv** [comment]
Thanks, I can confirm now it passes and I've sent the 2.0.11 for the distro integration.

**Comment by gaborbernat** [comment]
Hello, a fix for this issue has been released via virtualenv 20.0.11; see https://pypi.org/project/virtualenv/20.0.11/ (https://virtualenv.pypa.io/en/latest/changelog.html#v20-0-11-2020-03-18). Please give a try and report back if your issue has not been addressed; if not, please comment here, and we'll reopen the ticket. We want to apologize for the inconvenience this has caused you and say thanks for having patience while we resolve the unexpected bugs with this new major release.
![thanks](https://media.tenor.co/images/1114067881329bb4218880eeb7ec34fd/tenor.gif)
